extensions/objects/templates/algebraic-expression-editor.component.ts

```diff
@@ -56,7 +60,7 @@ export class AlgebraicExpressionEditorComponent implements OnInit {
   }
 
   ngOnInit(): void {
-    if (this.value === null) {
+    if (!this.value) {
```

Comment:
Why this change? The previous condition was more explicit. Now, this condition will hit even if the value is `''` or `undefined` or `false`. Is that expected?

Reply:
@iamprayush this.value can be undefined since it is an @input property and is initialized only after component interaction. So added a check for that, leaving the case where it is `''` or `false`

Reply:
To be more precise we can't assign a property of type string an undefined value see line 67. So we're adding a condition for it to be `''` if undefined

Reply:
as discussed with @vojtechjelinek ,Initializing this property at top with `''` ,so no condition to check for undefined

Reply:
I'm a little confused. If we're initializing it with '', then no change is required, right?